broker/src/main/java/io/zeebe/broker/system/SystemContext.java

```diff
@@ -33,6 +33,9 @@
       "Snapshot period %s needs to be larger then or equals to one minute.";
   private static final String MAX_BATCH_SIZE_ERROR_MSG =
       "Expected to have an append batch size maximum which is non negative and smaller then '%d', but was '%s'.";
+  private static final String REPLICATION_WITH_DISABLED_FLUSH_WARNING =
+      "Disabling explicit flushing with replication enabled is an experimental feature and can lead to inconsistencies "
+          + "in a lot of cases! Please refer to the documentation whether or not you should use this!";
```

Comment:
Documentation is not added yet. Would you do it in this PR or another?

Reply:
I opened an issue on the new documentation repo, as we shouldn't add docs in this repo anymore. See https://github.com/camunda-cloud/camunda-cloud-documentation/issues/38